crates/fuzzing/src/generators/table_ops.rs

```diff
@@ -3,6 +3,10 @@
 use arbitrary::Arbitrary;
 use std::fmt::Write;
```

Comment:
unused import

Reply:
If this is unused now feel free to delete!